features/managing_invoices/seeing_invoices_when_items_has_promitions_applied.feature

```diff
@@ -0,0 +1,24 @@
+@managing_invoices @ui
+Feature: Seeing information about an invoice with items has promotions applied
+    In order to see details of a specific invoice
+    As an Administrator
+    I want to be able to view basic information about an invoice
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a product "Angel T-Shirt" priced at "$10.00"
+        And the store allows shipping with "Galaxy Post"
+        And the store allows paying with "Space money"
+        And there is a promotion "Anatomy Park Promotion"
+        And this promotion gives "$1" off on every product with minimum price at "$5.00"
+        And there is a customer "rick.sanchez@wubba-lubba-dub-dub.com" that placed an order "#00000666"
+        And the customer "Lucifer Morningstar" addressed it to "Seaside Fwy", "90802" "Los Angeles" in the "United States"
+        And for the billing address of "Mazikeen Lilim" in the "Pacific Coast Hwy", "90806" "Los Angeles", "United States"
+        And the customer bought 2 "Angel T-Shirt" products
+        And the customer chose "Galaxy Post" shipping method to "United States" with "Space money" payment
+        And I am logged in as an administrator
+        And the order "#00000666" is already paid
+
+    Scenario: Seeing proper taxes on an invoice
```

Comment:
I believe you meant `discounts/promotions`... an in fact, they're not even there (2 lines below) 💃